src/applications/claims-status/components/ClaimsUnavailable.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function ClaimsUnavailable({ headerLevel = '4' }) {
+function ClaimsUnavailable({ headerLevel = 4 }) {
```

Comment:
Slipped this in here because all instances where this component is used have `headerLevel` as a number i.e. 2